jina/orchestrate/pods/helper.py

```diff
@@ -11,11 +14,49 @@
 
 if TYPE_CHECKING:  # pragma: no cover
     from argparse import Namespace
+    from pathlib import Path
+
+VCS_SCHEMES = [
```

Comment:
can't we just import all what is after this from hubble sdk ?

Reply:
We cannot guarantee that hubble-sdk will mantain this

Reply:
this is not hubble-sdk core and agreed with hubble that they should not care about installing from local